### ISSUE TITLE
why use ext_typoscript_setup.txt for enable = 1

### ISSUE THREAD

**Issue by ToGetATockenOnly** [a-r-m-i-n/min#7]
Hi,
the setting is surely to be object of configuration. But the ext_typoscript_setup.txt is hard to override. So why not drop the statement at that place to only be set elsewhere or position it in EXT:/min/Configuration/TypoScript/.. ?

Hint: All TS is loaded by EXT:sitepackage/ext_localconf.php or sitepackage/Configuration/TCA/Overrides/sys_template.php

**Comment by a-r-m-i-n** [comment]
Not sure which version you are looking at, but the latest one, does not use **ext_typoscript_setup.txt** anymore. 
This was a major change, when releasing version 2.0

**Comment by ToGetATockenOnly** [comment]
Yes, this is right. It is very obvious, but I was coming from a v1.9.0 version and did not recognize it right away. thanks.
